test/regression_test.sh

```diff
@@ -16,29 +16,12 @@ fi
 # refer to its dependencies with full, but with relative path
 # to the runfiles folder.
 SAVED_PWD=$(pwd)
-if [ "$TEST_TYPE" == "python" ]; then
-    script_content=$(cat <<EOF
-import os
-import runpy
-import sys
-sys.path.insert(0, "$SAVED_PWD")
-os.chdir("$SAVED_PWD")
-runpy.run_path("$TEST_NAME.$TEST_EXT", run_name="__main__")
-EOF
-    )
-else
-  script_content=$(cat <<EOF
-cd "$SAVED_PWD"
-source "$TEST_NAME.$TEST_EXT"
-EOF
-    )
-fi
 
 echo "Directory: ${PWD}"
 echo "Command:   $OPENROAD_EXE $ORD_ARGS -no_splash -no_init -exit  $TEST_NAME.$TEST_EXT > $LOG_FILE"
 
 cd ${BAZEL_SAVED_PWD:-$SAVED_PWD}
-$OPENROAD_EXE $ORD_ARGS -no_splash -no_init -exit <(echo "$script_content") &> $LOG_FILE
+$OPENROAD_EXE $ORD_ARGS -cd $SAVED_PWD -no_splash -no_init -exit $TEST_NAME.$TEST_EXT &> $LOG_FILE
 cd $SAVED_PWD
```

Comment:
why change to $SAVED_PWD here ? Looks like a leftover from before.

Reply:
if there is no bug now, don't do it as it might _mask_ a bug in waiting if there is. From the perspective of the shell script, you never changed the directory, only inside the process of openroad.

Reply:
Strike that: the cd $SAVED_PWD is necessary for the code below to work. The changing folder to launch openroad is an openroad launching concern that should not be narrowly scoped by a subshell to minimize surprises.